pkg/api-server/zones_ws.go

```diff
@@ -0,0 +1,63 @@
+package api_server
```

Comment:
this will be gone after switching to zones overview, right?

Reply:
Yes, this API will be gone